src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -67,6 +69,7 @@ trait ClusterComponent extends LeoComponent {
     def jupyterExtensionUri =         column[Option[String]]    ("jupyterExtensionUri",   O.Length(1024))
     def initBucket =                  column[String]            ("initBucket",            O.Length(1024))
     def serviceAccountKeyId =         column[Option[String]]    ("serviceAccountKeyId",   O.Length(254))
+    def stagingBucket =               column[Option[String]]    ("stagingBucket",         O.Length(254))
```

Comment:
The length is defined as 1024 in the liquibase script.

Probably 254 is reasonable, since this is just a bucket name, not a path.

Reply:
+1 consistent-ify